code/datalab-app/api.Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:7.10.0
+
+LABEL maintainer "joshua.foster@stfc.ac.uk"
+
+RUN mkdir -p /usr/src/api
+
+WORKDIR /usr/src/api
+
+COPY ./dist/api /usr/src/api
+
+COPY ./package.json /usr/src/api
+
+RUN npm install --production
```

Comment:
Will fix that. Just saw that hte Node.js Docker image comes with Yarn preinstalled.